R/chevron_tlg-S4methods.R

```diff
@@ -59,10 +63,15 @@ print_args <- function(run_call, additional_args, args, auto_pre = TRUE) {
   assert_class(run_call, "call")
   assert_list(args)
   assert_flag(auto_pre)
+
   run_call[[1]] <- NULL
   run_call <- as.list(run_call)
+
+  is_name <- Filter(is.name, run_call)
+  is_name["user_args"] <- NULL
+
   if (!is.null(additional_args)) {
-    run_call <- c(run_call[c("object", "adam_db")], additional_args)
+    run_call <- c(is_name, additional_args[setdiff(names(additional_args), names(is_name))])
```

Comment:
while this works, it is making the logic a bit complicated. Previously, `additional_args` will always overwrite the args in run_call. Now only those "names" will be replaced. It can be true that the provided argument is not a name, but some logical or string or int. then the printed argument is different from the provided.